tests/integration/test_syscollector/test_syscollector_configuration.py

```diff
@@ -0,0 +1,434 @@
+
+# Variables
+local_internal_options = {'wazuh_modules.debug': 2}
+if get_service() == 'wazuh-manager':
+    daemons_handler_configuration = {'daemons': [ANALYSISD_DAEMON, DB_DAEMON, MODULES_DAEMON], 'ignore_errors': True}
+elif sys.platform == 'win32':
+    daemons_handler_configuration = {'all_daemons': True, 'ignore_errors': True}
+    local_internal_options = {'windows.debug': 2}
+else:
+    daemons_handler_configuration = {'daemons': [MODULES_DAEMON], 'ignore_errors': True}
+
+# T1 Parameters
+t1_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector.yaml')
+t1_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_syscollector_deactivation.yaml')
+t1_config_parameters, t1_config_metadata, t1_case_ids = get_test_cases_data(t1_cases_path)
+t1_configurations = load_configuration_template(t1_config_path, t1_config_parameters, t1_config_metadata)
+
+# T2 Parameters
+t2_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector_scans_disabled.yaml')
+t2_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_all_scans_disabled.yaml')
+t2_config_parameters, t2_config_metadata, t2_case_ids = get_test_cases_data(t2_cases_path)
+t2_configurations = load_configuration_template(t2_config_path, t2_config_parameters, t2_config_metadata)
+
+# T3 Parameters
+t3_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector.yaml')
+t3_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_invalid_configurations.yaml')
+t3_config_parameters, t3_config_metadata, t3_case_ids = get_test_cases_data(t3_cases_path)
+t3_configurations = load_configuration_template(t3_config_path, t3_config_parameters, t3_config_metadata)
+
+# T4 Parameters
+t4_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector_no_tags.yaml')
+t4_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_default_values.yaml')
+t4_config_parameters, t4_config_metadata, t4_case_ids = get_test_cases_data(t4_cases_path)
+t4_configurations = load_configuration_template(t4_config_path, t4_config_parameters, t4_config_metadata)
+
+# T5 Parameters
+t5_config_path = os.path.join(CONFIGURATIONS_PATH, 'configuration_syscollector.yaml')
+t5_cases_path = os.path.join(TEST_CASES_PATH, 'case_test_scanning.yaml')
+t5_config_parameters, t5_config_metadata, t5_case_ids = get_test_cases_data(t5_cases_path)
+t5_config_metadata = t5_config_parameters
+t5_configurations = load_configuration_template(t5_config_path, t5_config_parameters, t5_config_metadata)
+
+
+# Tests
+@pytest.mark.parametrize('configuration, metadata', zip(t1_configurations, t1_config_metadata), ids=t1_case_ids)
+def test_syscollector_deactivation(configuration, metadata, set_wazuh_configuration,
+                                   configure_local_internal_options_module, truncate_monitored_files,
+                                   daemons_handler_function):
+    '''
+    description: Check that the module is disabled.
+
+    test_phases:
```

Comment:
Use the new `test_phases` documentation, please

Reply:
Done in 515b920